packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -136,17 +136,6 @@ module.exports = {
   module: {
     strictExportPresence: true,
     rules: [
-      // TODO: Disable require.ensure as it's not a standard language feature.
```

Comment:
You do not need to remove these three lines.  As far as I can see, they are not related to the tslint-loader block.  (Correct me if I am wrong.)  I am pointing it out because webpack.config.dev.js still has these three lines. :)

Reply:
Thanks for the hint - updated!